jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -130,6 +135,19 @@ protected DataSetReadyCallback getDataSetReadyCallback(final Integer startRange,
                 .withEmptyResultsCallback(() -> setEmptyResults());
     }
 
+    private Set<Long> findDuplicateId(List<TaskSummary> taskSummaries) {
+        Set<Long> id = new HashSet<>();
```

Comment:
Filter method is unneededm, you can remove the id set and the filter call. 
By set definition, the ids will be unique